src/App.js

```diff
@@ -522,7 +526,31 @@ export default class App extends React.Component<{}, AppState> {
         );
     }
 
+    componentDidMount() {
+        if (window.location.search != null) {
+            const query = new URLSearchParams(window.location.search);
+            const programQuery = query.get('program');
```

Comment:
Let's create a class to manage the URL params called `C2lcURLParams`:

- Basically a simple wrapper around `URLSearchParams`
- `constructor(query)`
    - Creates a `URLSearchParams` instance and stores in an instance var
    - Called like: `const params = new C2lcURLParams(window.location.search)`
- `getVersion()`
    - Returns `query.get('v')`
- `getProgram()`
    - Returns `query.get('p')`